galsim/chromatic.py

```diff
@@ -118,9 +118,10 @@ def __init__(self, obj):
         elif isinstance(obj, ChromaticObject):
             self.separable = obj.separable
             self.wave_list = obj.wave_list
-            if self.separable: self.SED = obj.SED
+            if self.separable:
+                self.SED = obj.SED
```

Comment:
Not a fan of this rule either, but I guess I can live with it.  Sometimes a one-liner is just nicer.